src/components/Cell/Cell.tsx

```diff
@@ -10,24 +10,31 @@ export interface CellProps<TData>
     extends Omit<React.TdHTMLAttributes<HTMLTableCellElement>, 'className'> {
     cell?: CellType<TData, unknown>;
     className?: string | ((cell?: CellType<TData, unknown>) => string);
+    attributes?:
+        | React.TdHTMLAttributes<HTMLTableCellElement>
+        | ((cell?: CellType<TData, unknown>) => React.TdHTMLAttributes<HTMLTableCellElement>);
 }
 
 export const Cell = <TData,>({
     cell,
     children,
     className: classNameProp,
     style,
+    attributes: getAttributes,
```

Comment:
Why do you think, that it's the better way for naming?
If you want to name it as the noun, then I would use something like `providedAttributes` to show that it's provided by the user

Reply:
Because there are other props named `className: classNameProp` and it's better to keep a common naming style. It doesn't matter if it would be `get*`, `provided*` or `*Prop`.

Reply:
Ok, I didn't notice that such way is used now